.github/workflows/bwc.yml

```diff
@@ -26,7 +26,6 @@ jobs:
         run: |
           echo "Running backwards compatibility tests ..."
           ./gradlew clean release -Dbuild.snapshot=true -x test -x IntegTest
-          ./gradlew mixedClusterTask --stacktrace
```

Comment:
Can we leave a note for now to modify the test once the 2.x end version is frozen in upstream?